hack/lib/util.sh

```diff
@@ -202,18 +202,6 @@ kube::util::gen-docs() {
 
   mkdir -p "${dest}/docs/user-guide/kubectl/"
   "${gendocs}" "${dest}/docs/user-guide/kubectl/"
-  mkdir -p "${dest}/docs/admin/"
-  "${genkubedocs}" "${dest}/docs/admin/" "kube-apiserver"
```

Comment:
can we get redi of the definitions of genkubedocs and genfeddocs?  How about those binaries?